draft-ietf-rats-architecture.md

```diff
@@ -1424,30 +1425,57 @@ in terms of `time(RX_v)-time(RG_v)`, then the Relying Party can check
 
 ## Example 3: Handle-based Passport Model Example
 
-Handles are a third option to establish time-keeping next to nonces or timestamps.
-Handles are opaque data intended to be available to all RATS roles that interact with each other, such as the Attester or Verifier, in specified intervals.
-To enable this availability, handles are distributed centrally by the Handle Distributor role over the network.
-As any other role, the Handle Distributor role can be taken on by a dedicated entity or collapsed with other roles, such as a Verifier.
-The use of handles can compensate for a lack of clocks or other sources of time on entities taking on RATS roles.
-The only entity that requires access to a source of time is the entity taking on the role of Handle Distributor.
+A third option to establish the epoch of Evidence or Attestation Result
+generation - next to nonces and timestamps ({{freshness}}) - is by using
+"handles".
 
-Handles are different from nonces as they can be used more than once and can be used by more than one entity at the same time.
-Handles are different from timestamps as they do not have to convey information about a point in time, but their reception creates that information.
-The reception of a handle is similar to the event that increments a relative tickcounter.
-Receipt of a new handle invalidates a previously received handle.
+Handles are opaque, non-predictable data regularly broadcast by a Handle Distributor.  
+Each handle represents a specific epoch,
+in which generated Evidence can be asserted to be fresh.
```

Comment:
The freshness section still calls them nonces in lines 1109-1114. We should be consistent, maybe by calling them handles there.   And these two sentences are not about the example per se, they're about the definition of handles and so I think they should move to where line 1109-1114 are.

Reply:
Dave took on the task to generate text in the Freshness section that introduces the Handle concept in the body of the document.

Reply:
Recommend rebasing on top of PR #257